src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesByPrisonerMigrationService.kt

```diff
@@ -86,6 +85,7 @@ class CaseNotesByPrisonerMigrationService(
             caseNote.caseNoteType,
             caseNote.caseNoteSubType,
             caseNote.authorStaffId,
+            caseNote.amendments,
```

Comment:
I wasnt expecting this to work but it does successfully group case notes which have distinct lists containing distinct amendment objects containing identical field values !